crypto/asn1/i2d_evp.c

```diff
@@ -102,9 +122,13 @@ int i2d_PrivateKey(const EVP_PKEY *a, unsigned char **pp)
 int i2d_PublicKey(const EVP_PKEY *a, unsigned char **pp)
 {
     if (evp_pkey_is_provided(a)) {
-        const char *output_structures[] = { "type-specific", NULL };
+        static const struct type_and_structure_st output_info[] = {
+            { "DER", "type-specific" },
+            { "blob", NULL },    /* for EC */
```

Comment:
Is there much chance we'd ever want to add a 'structure' for things of type 'blob'?  This is only i2d_PublicKey(), after all, and IIRC you could call that on something with a private key as well, so we might need to indicate which subset of stuff to encode.

Reply:
The output structure isn't there to differentiate between different content subsets.  That's what the accepted selection bits do.

Reply:
A "blob" is, more or less by logical definition, always be type specific.  So yeah sure, we could use "type-specific", but that would just be redundant.